docs/intro/overrides.rst

```diff
@@ -0,0 +1,349 @@
+    - Your project's objective is to handle as much eCommerce websites as you
+      can.
+
+        - Thus, you'd want to use the already available packages above and
+          perhaps improve on them or create new Page Objects for new websites.
+
+Remember that all of the :class:`~.OverrideRule` are declared by annotating
+Page Objects using the :func:`web_poet.handle_urls` via ``@handle_urls``. Thus,
+they can easily be accessed using the :meth:`~.PageObjectRegistry.get_overrides`
+of ``web_poet.default_registry``.
+
+This can be done something like:
+
+.. code-block:: python
+
+    from web_poet import default_registry, consume_modules
+
+    # ❌ Remember that this wouldn't retrieve any rules at all since the
+    # annotations are NOT properly loaded.
+    rules = default_registry.get_overrides()
+    print(rules)  # []
+
+    # ✅ Instead, you need to run the following so that all of the Page
+    # Objects in the external packages are recursively loaded.
+    consume_modules("ecommerce_page_objects", "gadget_sites_page_objects")
+    rules = default_registry.get_overrides()
+
+    # Alternatively, this could be shortened using:
+    rules = default_registry.get_overrides(consume=["ecommerce_page_objects", "gadget_sites_page_objects"])
+
+    # The collected rules would then be as follows:
+    print(rules)
+    # 1. OverrideRule(for_patterns=Patterns(include=['site_1.com'], exclude=[], priority=500), use=<class 'ecommerce_page_objects.site_1.EcomSite1'>, instead_of=<class 'ecommerce_page_objects.EcomGenericPage'>, meta={})
+    # 2. OverrideRule(for_patterns=Patterns(include=['site_2.com'], exclude=[], priority=500), use=<class 'ecommerce_page_objects.site_2.EcomSite2'>, instead_of=<class 'ecommerce_page_objects.EcomGenericPage'>, meta={})
+    # 3. OverrideRule(for_patterns=Patterns(include=['site_2.com'], exclude=[], priority=500), use=<class 'gadget_sites_page_objects.site_2.GadgetSite2'>, instead_of=<class 'gadget_sites_page_objects.GadgetGenericPage'>, meta={})
+    # 4. OverrideRule(for_patterns=Patterns(include=['site_3.com'], exclude=[], priority=500), use=<class 'gadget_sites_page_objects.site_3.GadgetSite3'>, instead_of=<class 'gadget_sites_page_objects.GadgetGenericPage'>, meta={})
+
+.. note::
+
+    Once :func:`~.web_poet.overrides.consume_modules` is called, then all
+    external Page Objects are loaded and available for the entire runtime
+    duration. Calling :func:`~.web_poet.overrides.consume_modules` again makes
+    no difference unless a new set of modules are provided.
+
+.. _`intro-rule-subset`:
+
+Using only a subset of the available OverrideRules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Suppose that the use case from the previous section has changed wherein a
+subset of :class:`~.OverrideRule` would be used. This could be achieved by
+using the :meth:`~.PageObjectRegistry.search_overrides` method which allows for
+convenient selection of a subset of rules from the ``default_registry``.
```

Comment:
```suggestion
convenient selection of a subset of rules from the registry.
```

Reply:
Updated this on https://github.com/scrapinghub/web-poet/pull/27/commits/37e5ba1a3696f629801478fe78ea1729164f39df